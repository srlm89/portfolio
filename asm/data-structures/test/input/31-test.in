ERASE actual/31-test.out
INSERT 26 2.638590 Green Lantern First Flight   720p
PRINT_IN_ORDER actual/31-test.out
INSERT 16 1.212610 Terminator Salvation Director's cut  ( 2009)    1080p
INSERT 7 0.581110 Alvin And The Chipmunks [720p]
REMOVE_ID 26
PRINT_IN_ORDER actual/31-test.out
INSERT 5 1.241176 James Bond 16 - Licence to Kill [1989]
INSERT 8 2.231378 Exiled   720p
INSERT 20 0.239363 The X Files - I Want To Believe    720p
REMOVE_ID 5
PRINT_IN_ORDER actual/31-test.out
INSERT 19 2.489137 The Dark Knight [2008]   1080p
INSERT 32 8.759857 Rocky 3 [1982] [BluRay] [720p] [AC3]
INSERT 27 9.129272 Superhero Movie   1080p
INSERT 28 9.599550 Shutter   720p
REMOVE_ID 8
REMOVE_ID 7
PRINT_IN_ORDER actual/31-test.out
INSERT 25 9.483939 Apocalypto   720p
INSERT 9 9.553903 Mr. & Mrs. Smith   720p
INSERT 14 8.133470 The Mummy 1-2   1080p
INSERT 15 5.316349 SWAT    720p
INSERT 18 5.698036 Black Hawk Down   1080p
REMOVE_ID 14
REMOVE_ID 20
PRINT_IN_ORDER actual/31-test.out
INSERT 21 5.341553 Jurassic Park 1-3  1080p
INSERT 33 8.268683 Fast and the Furious 4 - 1080p
INSERT 30 8.962349 SAW V
INSERT 40 3.510152 Bride Wars   720p
INSERT 39 4.466133 Apocalypto   720p
INSERT 49 1.743422 Oceans.Thirteen.2007.720p.HDDVD.x264-SiNNERS
REMOVE_ID 30
REMOVE_ID 33
REMOVE_ID 28
PRINT_IN_ORDER actual/31-test.out
INSERT 46 2.744007 Seven Swords    1080p
INSERT 34 7.484050 Constantine 720p
INSERT 11 2.392086 Push    1080p
INSERT 37 8.220311 Music.And.Lyrics.720p.HDDVD.x264-REVEiLLE
INSERT 3 3.644718 Madagascar 1 [2005] [1080p]
INSERT 2 9.265292 Eurotrip 720p
INSERT 55 1.968791 Terminator 3 Rise Of The Machine [2003]   720p
REMOVE_ID 16
REMOVE_ID 40
REMOVE_ID 9
PRINT_IN_ORDER actual/31-test.out
INSERT 10 3.065239 James Bond 16 - Licence to Kill [1989]
INSERT 0 5.257406 Red Dragon    1080p
INSERT 56 2.907616 Inkheart [2008] [720p]
INSERT 57 3.017352 The Dark Knight [2008]   1080p
INSERT 44 6.551820 Species   720p
INSERT 29 3.724233 My Bloody Valentine 3D    1080p
INSERT 43 3.623755 007 Licence to Kill [1989] [BluRay] [720p] [DTS]
INSERT 62 6.959245 Push    1080p
REMOVE_ID 27
REMOVE_ID 62
REMOVE_ID 46
REMOVE_ID 15
PRINT_IN_ORDER actual/31-test.out
INSERT 67 4.150264 WALL-E.720p.BluRay.x264-iNFAMOUS
INSERT 35 3.671181 A.Beautiful.Mind.2001.720p.HDTV.x264-S26
INSERT 17 6.450099 The Haunting In Connecticut [2009] [720p]
INSERT 22 0.499377 Seven Swords    1080p
INSERT 71 4.302039 Fast and the Furious 4 - 1080p
INSERT 36 2.760108 Scooby.Doo.The.Mystery.Begins.2009   720p
INSERT 6 9.450109 Leaving Las Vegas  720p
INSERT 24 5.721049 300   720p
INSERT 42 4.439644 Exiled   720p
REMOVE_ID 42
REMOVE_ID 67
REMOVE_ID 2
REMOVE_ID 19
PRINT_IN_ORDER actual/31-test.out
INSERT 45 7.609995 Pitch Black [720p]
INSERT 53 9.548183 13.Going.on.30.2004.720p.BluRay.x264-METiS_tehPARADOX.COM
INSERT 4 2.909295 Hitman    720p
INSERT 61 3.506811 Into The Blue 720P
INSERT 51 7.401404 Miss March [720p]
INSERT 12 9.528586 Sky.Captain.And.The.World.Of.Tomorrow[www.x1949x.com]    720p
INSERT 65 5.897732 Twilight [2008]   720p
INSERT 69 6.108974 Click   1080p
INSERT 50 3.500136 X-Men 3   1080p
INSERT 70 4.375201 Constantine 720p
REMOVE_ID 24
REMOVE_ID 37
REMOVE_ID 0
REMOVE_ID 55
REMOVE_ID 3
PRINT_IN_ORDER actual/31-test.out
INSERT 79 6.008495 Danny the Dog   1080p
INSERT 13 5.011777 The Secret of Moonarce [720p]
INSERT 73 6.996727 Oceans.Twelve.2004.720p.HDDVD.x264-CDDHD
INSERT 52 9.319222 The Sniper 720p
INSERT 64 0.151166 Meet The Fockers    720p
INSERT 90 5.938673 Into The Blue 720P
INSERT 80 3.765479 American Beauty   720p
INSERT 91 8.641921 Back.to.the.Future.Part.III
INSERT 31 3.729407 PIXAR Movie
INSERT 82 3.057144 Demo (Sony, Sharp, THX and more)
INSERT 47 1.088938 A Bug's Life [720p]
REMOVE_ID 49
REMOVE_ID 43
REMOVE_ID 80
REMOVE_ID 45
REMOVE_ID 4
PRINT_IN_ORDER actual/31-test.out
