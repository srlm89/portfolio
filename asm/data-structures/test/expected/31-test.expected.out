[ 26 2.638590 Green Lantern First Flight   720p ]
[ 7 0.581110 Alvin And The Chipmunks [720p] ][ 16 1.212610 Terminator Salvation Director's cut  ( 2009)    1080p ]
[ 7 0.581110 Alvin And The Chipmunks [720p] ][ 8 2.231378 Exiled   720p ][ 16 1.212610 Terminator Salvation Director's cut  ( 2009)    1080p ][ 20 0.239363 The X Files - I Want To Believe    720p ]
[ 16 1.212610 Terminator Salvation Director's cut  ( 2009)    1080p ][ 19 2.489137 The Dark Knight [2008]   1080p ][ 20 0.239363 The X Files - I Want To Believe    720p ][ 27 9.129272 Superhero Movie   1080p ][ 28 9.599550 Shutter   720p ][ 32 8.759857 Rocky 3 [1982] [BluRay] [720p] [AC3] ]
[ 9 9.553903 Mr. & Mrs. Smith   720p ][ 15 5.316349 SWAT    720p ][ 16 1.212610 Terminator Salvation Director's cut  ( 2009)    1080p ][ 18 5.698036 Black Hawk Down   1080p ][ 19 2.489137 The Dark Knight [2008]   1080p ][ 25 9.483939 Apocalypto   720p ][ 27 9.129272 Superhero Movie   1080p ][ 28 9.599550 Shutter   720p ][ 32 8.759857 Rocky 3 [1982] [BluRay] [720p] [AC3] ]
[ 9 9.553903 Mr. & Mrs. Smith   720p ][ 15 5.316349 SWAT    720p ][ 16 1.212610 Terminator Salvation Director's cut  ( 2009)    1080p ][ 18 5.698036 Black Hawk Down   1080p ][ 19 2.489137 The Dark Knight [2008]   1080p ][ 21 5.341553 Jurassic Park 1-3  1080p ][ 25 9.483939 Apocalypto   720p ][ 27 9.129272 Superhero Movie   1080p ][ 32 8.759857 Rocky 3 [1982] [BluRay] [720p] [AC3] ][ 39 4.466133 Apocalypto   720p ][ 40 3.510152 Bride Wars   720p ][ 49 1.743422 Oceans.Thirteen.2007.720p.HDDVD.x264-SiNNERS ]
[ 2 9.265292 Eurotrip 720p ][ 3 3.644718 Madagascar 1 [2005] [1080p] ][ 11 2.392086 Push    1080p ][ 15 5.316349 SWAT    720p ][ 18 5.698036 Black Hawk Down   1080p ][ 19 2.489137 The Dark Knight [2008]   1080p ][ 21 5.341553 Jurassic Park 1-3  1080p ][ 25 9.483939 Apocalypto   720p ][ 27 9.129272 Superhero Movie   1080p ][ 32 8.759857 Rocky 3 [1982] [BluRay] [720p] [AC3] ][ 34 7.484050 Constantine 720p ][ 37 8.220311 Music.And.Lyrics.720p.HDDVD.x264-REVEiLLE ][ 39 4.466133 Apocalypto   720p ][ 46 2.744007 Seven Swords    1080p ][ 49 1.743422 Oceans.Thirteen.2007.720p.HDDVD.x264-SiNNERS ][ 55 1.968791 Terminator 3 Rise Of The Machine [2003]   720p ]
[ 0 5.257406 Red Dragon    1080p ][ 2 9.265292 Eurotrip 720p ][ 3 3.644718 Madagascar 1 [2005] [1080p] ][ 10 3.065239 James Bond 16 - Licence to Kill [1989] ][ 11 2.392086 Push    1080p ][ 18 5.698036 Black Hawk Down   1080p ][ 19 2.489137 The Dark Knight [2008]   1080p ][ 21 5.341553 Jurassic Park 1-3  1080p ][ 25 9.483939 Apocalypto   720p ][ 29 3.724233 My Bloody Valentine 3D    1080p ][ 32 8.759857 Rocky 3 [1982] [BluRay] [720p] [AC3] ][ 34 7.484050 Constantine 720p ][ 37 8.220311 Music.And.Lyrics.720p.HDDVD.x264-REVEiLLE ][ 39 4.466133 Apocalypto   720p ][ 43 3.623755 007 Licence to Kill [1989] [BluRay] [720p] [DTS] ][ 44 6.551820 Species   720p ][ 49 1.743422 Oceans.Thirteen.2007.720p.HDDVD.x264-SiNNERS ][ 55 1.968791 Terminator 3 Rise Of The Machine [2003]   720p ][ 56 2.907616 Inkheart [2008] [720p] ][ 57 3.017352 The Dark Knight [2008]   1080p ]
[ 0 5.257406 Red Dragon    1080p ][ 3 3.644718 Madagascar 1 [2005] [1080p] ][ 6 9.450109 Leaving Las Vegas  720p ][ 10 3.065239 James Bond 16 - Licence to Kill [1989] ][ 11 2.392086 Push    1080p ][ 17 6.450099 The Haunting In Connecticut [2009] [720p] ][ 18 5.698036 Black Hawk Down   1080p ][ 21 5.341553 Jurassic Park 1-3  1080p ][ 22 0.499377 Seven Swords    1080p ][ 24 5.721049 300   720p ][ 25 9.483939 Apocalypto   720p ][ 29 3.724233 My Bloody Valentine 3D    1080p ][ 32 8.759857 Rocky 3 [1982] [BluRay] [720p] [AC3] ][ 34 7.484050 Constantine 720p ][ 35 3.671181 A.Beautiful.Mind.2001.720p.HDTV.x264-S26 ][ 36 2.760108 Scooby.Doo.The.Mystery.Begins.2009   720p ][ 37 8.220311 Music.And.Lyrics.720p.HDDVD.x264-REVEiLLE ][ 39 4.466133 Apocalypto   720p ][ 43 3.623755 007 Licence to Kill [1989] [BluRay] [720p] [DTS] ][ 44 6.551820 Species   720p ][ 49 1.743422 Oceans.Thirteen.2007.720p.HDDVD.x264-SiNNERS ][ 55 1.968791 Terminator 3 Rise Of The Machine [2003]   720p ][ 56 2.907616 Inkheart [2008] [720p] ][ 57 3.017352 The Dark Knight [2008]   1080p ][ 71 4.302039 Fast and the Furious 4 - 1080p ]
[ 4 2.909295 Hitman    720p ][ 6 9.450109 Leaving Las Vegas  720p ][ 10 3.065239 James Bond 16 - Licence to Kill [1989] ][ 11 2.392086 Push    1080p ][ 12 9.528586 Sky.Captain.And.The.World.Of.Tomorrow[www.x1949x.com]    720p ][ 17 6.450099 The Haunting In Connecticut [2009] [720p] ][ 18 5.698036 Black Hawk Down   1080p ][ 21 5.341553 Jurassic Park 1-3  1080p ][ 22 0.499377 Seven Swords    1080p ][ 25 9.483939 Apocalypto   720p ][ 29 3.724233 My Bloody Valentine 3D    1080p ][ 32 8.759857 Rocky 3 [1982] [BluRay] [720p] [AC3] ][ 34 7.484050 Constantine 720p ][ 35 3.671181 A.Beautiful.Mind.2001.720p.HDTV.x264-S26 ][ 36 2.760108 Scooby.Doo.The.Mystery.Begins.2009   720p ][ 39 4.466133 Apocalypto   720p ][ 43 3.623755 007 Licence to Kill [1989] [BluRay] [720p] [DTS] ][ 44 6.551820 Species   720p ][ 45 7.609995 Pitch Black [720p] ][ 49 1.743422 Oceans.Thirteen.2007.720p.HDDVD.x264-SiNNERS ][ 50 3.500136 X-Men 3   1080p ][ 51 7.401404 Miss March [720p] ][ 53 9.548183 13.Going.on.30.2004.720p.BluRay.x264-METiS_tehPARADOX.COM ][ 56 2.907616 Inkheart [2008] [720p] ][ 57 3.017352 The Dark Knight [2008]   1080p ][ 61 3.506811 Into The Blue 720P ][ 65 5.897732 Twilight [2008]   720p ][ 69 6.108974 Click   1080p ][ 70 4.375201 Constantine 720p ][ 71 4.302039 Fast and the Furious 4 - 1080p ]
[ 6 9.450109 Leaving Las Vegas  720p ][ 10 3.065239 James Bond 16 - Licence to Kill [1989] ][ 11 2.392086 Push    1080p ][ 12 9.528586 Sky.Captain.And.The.World.Of.Tomorrow[www.x1949x.com]    720p ][ 13 5.011777 The Secret of Moonarce [720p] ][ 17 6.450099 The Haunting In Connecticut [2009] [720p] ][ 18 5.698036 Black Hawk Down   1080p ][ 21 5.341553 Jurassic Park 1-3  1080p ][ 22 0.499377 Seven Swords    1080p ][ 25 9.483939 Apocalypto   720p ][ 29 3.724233 My Bloody Valentine 3D    1080p ][ 31 3.729407 PIXAR Movie ][ 32 8.759857 Rocky 3 [1982] [BluRay] [720p] [AC3] ][ 34 7.484050 Constantine 720p ][ 35 3.671181 A.Beautiful.Mind.2001.720p.HDTV.x264-S26 ][ 36 2.760108 Scooby.Doo.The.Mystery.Begins.2009   720p ][ 39 4.466133 Apocalypto   720p ][ 44 6.551820 Species   720p ][ 47 1.088938 A Bug's Life [720p] ][ 50 3.500136 X-Men 3   1080p ][ 51 7.401404 Miss March [720p] ][ 52 9.319222 The Sniper 720p ][ 53 9.548183 13.Going.on.30.2004.720p.BluRay.x264-METiS_tehPARADOX.COM ][ 56 2.907616 Inkheart [2008] [720p] ][ 57 3.017352 The Dark Knight [2008]   1080p ][ 61 3.506811 Into The Blue 720P ][ 64 0.151166 Meet The Fockers    720p ][ 65 5.897732 Twilight [2008]   720p ][ 69 6.108974 Click   1080p ][ 70 4.375201 Constantine 720p ][ 71 4.302039 Fast and the Furious 4 - 1080p ][ 73 6.996727 Oceans.Twelve.2004.720p.HDDVD.x264-CDDHD ][ 79 6.008495 Danny the Dog   1080p ][ 82 3.057144 Demo (Sony, Sharp, THX and more) ][ 90 5.938673 Into The Blue 720P ][ 91 8.641921 Back.to.the.Future.Part.III ]
